### ISSUE TITLE
Mini Play opens, and starts to grow bigger and bigger

### ISSUE THREAD

**Issue by moymike** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2471]
Windows 10, Version 1703 (OS Build 15063.250)
Google Play Music Desktop Player, Version: 4.3.0

Mini Play opens, and starts to grow bigger and bigger, play, pause.. no keys or buttons have any effect, it just keeps growing.

I rebooted, no change.
I uninstalled, rebooted, reinstalled, no change.

When I click the bottom right icon, "Show mini player", the player goes mini, then proceeds to grow non-stop.  Within a minute or two it exceeds the size of my entire screen and still continues to grow.

**Comment by MarshallOfSound** [comment]
Duplicate, please check existing issues before opening new ones

**Comment by moymike** [comment]
I did look through the open issues and found nothing similar.  I looked through the closed issues after you closed my issue, and found a few postings, but nothing that pointed me toward any resolution or cause.

While I appreciate the speed at which you looked at my issue, a link to the issue that mine duplicates would have maybe ended my search for a resolution.  I am experiencing this issue on a new windows 10 machine with very little else installed.
